FBAllocationTracker/NSObject+FBAllocationTracker.mm

```diff
@@ -46,3 +48,31 @@ - (void)fb_newDealloc
 @end
 
 #endif // _INTERNAL_FBAT_ENABLED
+
+
+bool fb_isZombieObject(_Nullable id object) {
+  if (object == nil) { return true; }
+  if (!fb_isNSZombieEnabled()) { return false; }
+
+  const char *className = object_getClassName(object);
+  static const char *zombiePrefix = "_NSZombie";
+  return strncmp(className, zombiePrefix, strlen(zombiePrefix)) == 0;
```

Comment:
Yeah, good point, this method will be called a lot. Will add this change as well